sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalogSuite.scala

```diff
@@ -107,6 +111,41 @@ class JDBCTableCatalogSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("simple scan") {
+    checkAnswer(sql("SELECT name, id FROM h2.test.people"), Seq(Row("fred", 1), Row("mary", 2)))
+  }
+
+  test("scan with filter push-down") {
```

Comment:
Just in case, filters pushdown to JDBC isn't under a config. There are config for all builtin datasources
- spark.sql.parquet.filterPushdown
- spark.sql.orc.filterPushdown
- spark.sql.csv.filterPushdown.enabled
- spark.sql.json.filterPushdown.enabled
- spark.sql.avro.filterPushdown.enabled
except JDBC?

Reply:
Seems JDBC uses `JDBCOptions.pushDownPredicate` to control if push down the filter.